packages/react-aria-components/docs/DropZone.mdx

```diff
@@ -256,19 +256,28 @@ To allow the selection of files from the user's device, pass `FileTrigger` as a
 import {FileTrigger, Button} from 'react-aria-components';
 
 function Example() {
-  let [file, setFile] = React.useState(null);
+  let [files, setFiles] = React.useState(null);
 
   return(
-    <DropZone>
+    <DropZone
+      onDrop={(e) => {
+        let files = e.items;
+        let urls = files.map((file) => file.name);
+        setFiles(urls.join(', '));
+      }}>
       <FileTrigger
+        allowsMultiple
         onSelect={(e) => {
           let files = Array.from(e);
           let urls = files.map((file) => file.name);
-          setFile(urls);
+          setFiles(urls.join(', '));
         }}>
-        <Button>Select a file</Button>
+        <Button>Select files</Button>
       </FileTrigger>
-      {file && file}
+      <br/>
```

Comment:
```suggestion
```
I think we can get rid of this and in the [section](https://github.com/adobe/react-spectrum/blob/bb3f54fd93e36cc99d707c4314e13b93606bc3e8/packages/react-aria-components/docs/DropZone.mdx#L280) below this code there is a css section where you can add `.react-aria-Text { display: 'inline-block' }`.